q_yaar_platform/common/constants.py

```diff
@@ -0,0 +1,3 @@
+class Length:
```

Comment:
It's always better to store constants nicely vs having random numbers float around the codebase.